rest/model/version.go

```diff
@@ -45,9 +45,12 @@ type APIVersion struct {
 	Requester *string   `json:"requester"`
 	Errors    []*string `json:"errors"`
 	// Will be null for versions created before this field was added.
-	Activated *bool       `json:"activated"`
-	Aborted   *bool       `json:"aborted"`
-	GitTags   []APIGitTag `json:"git_tags"`
+	Activated *bool `json:"activated"`
+	Aborted   *bool `json:"aborted"`
+	// The git tag that triggered this version, if any.
+	TriggeredGitTag *APIGitTag `json:"triggered_by_git_tag"`
+	// The pushed git tags to this version.
+	GitTags []APIGitTag `json:"git_tags"`
```

Comment:
nit: can we say "Git tags that were pushed to this version" or something, to be a bit more clear?